openprescribing/api/views_spending.py

```diff
@@ -359,24 +359,23 @@ def tariff(request, format=None):
 def spending_by_ccg(request, format=None):
     codes = utils.param_to_list(request.query_params.get('code', []))
     codes = utils.get_bnf_codes_from_number_str(codes)
-    orgs = utils.param_to_list(request.query_params.get('org', []))
+    pct_ids = utils.param_to_list(request.query_params.get('org', []))
 
     spending_type = utils.get_spending_type(codes)
     if spending_type is False:
         err = CODE_LENGTH_ERROR
         return Response(err, status=400)
 
-    if not spending_type or spending_type == 'bnf-section' \
-       or spending_type == 'chemical':
-        query = _get_query_for_chemicals_or_sections_by_ccg(codes, orgs,
-                                                            spending_type)
+    if spending_type in ['product', 'presentation']:
```

Comment:
This is a good change but might be even better if we enumerate all the possible spending types as constants somewhere, so they can all be seen in one place, and no string typos possible?